core/common/src/main/java/alluxio/wire/MasterWebUIMetrics.java

```diff
@@ -31,9 +31,9 @@
   private int mMasterCapacityUsedPercentage;
   private int mMasterUnderfsCapacityFreePercentage;
   private int mMasterUnderfsCapacityUsedPercentage;
-  private Map<String, Counter> mRpcInvocationMetrics;
+  private Map<String, Long> mRpcInvocationMetrics;
   private Map<String, Map<String, Long>> mUfsOps;
-  private Map<String, Metric> mOperationMetrics;
+  private Map<String, UIMetric> mOperationMetrics;
```

Comment:
I think this could just be a `Long`, similar to `mRpcInvocationMetrics`

Reply:
I tried, but the jackson serializer refused to cooperate